### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by ghost**
```
jax/docs/conf.py:104:0: "napolean" is a misspelling of "napoleon"
jax/docs/notebooks/JAX_pytrees.ipynb:264:57: "dictionay" is a misspelling of "dictionary"
jax/docs/notebooks/score_matching.ipynb:194:55: "dimention" is a misspelling of "dimension"
jax/docs/notebooks/score_matching.ipynb:395:406: "Teh" is a misspelling of "The"
jax/docs/notebooks/score_matching.ipynb:499:194: "dimentional" is a misspelling of "dimensional"
jax/jax/experimental/vectorize.py:238:11: "abritrary" is a misspelling of "arbitrary"
jax/jax/tools/jax_to_hlo.py:53:41: "primarly" is a misspelling of "primarily"
```

### CODE REVIEW

**Comment by mattjj** [comment]
Woohoo awesome! But the jax/docs/conf.py one isn't a typo. Could you remove that one?

File: docs/conf.py [hunk 1]
```diff
@@ -101,7 +101,7 @@
 
 
 autosummary_generate = True
-napolean_use_rtype = False
+napoleon_use_rtype = False
```
Comment (yoverflow) [inline docs/conf.py]:
```suggestion
napolean_use_rtype = False
```

**Comment by yoverflow** [comment]
@googlebot I fixed it.

**Comment by yoverflow** [comment]
@mattjj Fixed! Can you review it now?

**Comment by mattjj** [comment]
Fantastic, thank you for this contribution!

**Pull request merged**